packages/atomic/src/components/ipx/atomic-ipx-button/atomic-ipx-button.pcss

```diff
@@ -0,0 +1,67 @@
+@import '../../../global/global.pcss';
```

Comment:
It would be best to leverage our [Tailwind](https://tailwindcss.com/) lib instead of adding custom CSS in this file

Reply:
I made some changes to use Tailwind.